app/src/main/java/io/github/trojan_gfw/igniter/servers/fragment/ServerListFragment.java

```diff
@@ -312,6 +314,9 @@ public boolean onOptionsItemSelected(MenuItem item) {
             case R.id.action_subscribe_servers:
                 mPresenter.updateSubscribeServers();
                 return true;
+            case R.id.action_test_all_proxy_server:
+                Log.d(TAG, "test action_test_all_proxy_server");
+                mPresenter.pingAllProxyServer(mServerListAdapter.getData());
```

Comment:
Please return `true` to consume the menu process.

Reply:
Ok.